### PR TITLE
[Platform] Add metadata to variant page

### PR DESCRIPTION
**Pull request by gjmcn**
## Description

Added profile to variants page with metadata.

**Issue:** https://github.com/opentargets/issues/issues/3299

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Load variant page with dev server (e.g. http://localhost:3000/variant/20_31257670_C_T).

## Checklist:

- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by chinmehta — CHANGES_REQUESTED** [review]

File: apps/platform/src/pages/VariantPage/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { ProfileHeader as BaseProfileHeader, Field } from "ui";
```
[the remark this answers is not in the document]
Comment (gjmcn) [reply apps/platform/src/pages/VariantPage/Profile.tsx]: Pushed.

File: apps/platform/src/pages/VariantPage/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+function Profile({ metadata }: ProfileProps) {
+  
+  // always set loading to false for now
+  const loading = false;
+
+  return (
+    <BaseProfileHeader>
+      
+      <>
```
[the remark this answers is not in the document]
Comment (gjmcn) [reply apps/platform/src/pages/VariantPage/Profile.tsx]: Pushed

File: apps/platform/src/pages/VariantPage/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        <Typography variant="subtitle1" mt={0}>Location</Typography>
+        <Field loading={loading} title="GRCh38">
+          {`${metadata.chromosome}:${metadata.position}`}
```
[the remark this answers is not in the document]
Comment (gjmcn) [reply apps/platform/src/pages/VariantPage/Profile.tsx]: Pushed.

**Comment by gjmcn** [comment]
Replaced by PR #376

**Pull request closed without merging**